### PR TITLE
fix (docs): correct error behavior description on fullStream.

### PR DESCRIPTION
**Pull request by lgrammel**

### CODE REVIEW

File: content/docs/07-reference/ai-sdk-core/02-stream-text.mdx [hunk 1]
```diff
@@ -553,7 +553,7 @@ for await (const textPart of textStream) {
       name: 'fullStream',
       type: 'AsyncIterable<TextStreamPart> & ReadableStream<TextStreamPart>',
       description:
-        'A stream with all events, including text deltas, tool calls, tool results, and errors. You can use it as either an AsyncIterable or a ReadableStream. When an error occurs, the stream will throw the error.',
+        'A stream with all events, including text deltas, tool calls, tool results, and errors. You can use it as either an AsyncIterable or a ReadableStream. Only errors that break the break the stream are thrown.',
```
Comment (wong2) [inline content/docs/07-reference/ai-sdk-core/02-stream-text.mdx:556]: Is `break the break` typo here?

Comment (lgrammel) [reply content/docs/07-reference/ai-sdk-core/02-stream-text.mdx:556]: good catch!

**Comment by wong2** [comment]
What kind of error will break the stream, and what won't?

**Comment by lgrammel** [comment]
> What kind of error will break the stream, and what won't?

E.g. network errors where the stream is stopped will be thrown. Error such as incorrectly formatted stream parts or provider error parts will be converted.

**Comment by wong2** [comment]
Thanks! LGTM

**Review by nicoalbanese — APPROVED** [review]

**Pull request merged**